layouts/filter.tsx

```diff
@@ -8,8 +8,7 @@ import FilterHeader from "components/molecules/FilterHeader/filter-header";
 
 import useNav from "lib/hooks/useNav";
 
-const FilterLayout = ({children}: {children: React.ReactNode}) => {
-
+const FilterLayout = ({ children }: { children: React.ReactNode }) => {
```

Comment:
nit: revert unnecessary changes to this file

Reply:
this has been done.